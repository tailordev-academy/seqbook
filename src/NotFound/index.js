/* @flow */
import React from 'react';

const NotFound = () => (
  <div className="NotFound">
    <div className="container-fluid">
      <h1>Page Not Found</h1>
    </div>
  </div>
);

export default NotFound;
