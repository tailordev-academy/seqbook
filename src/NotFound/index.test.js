import React from 'react';
import renderer from 'react-test-renderer';

import NotFound from './index';

it('renders correctly', () => {
  const tree = renderer.create(
    <NotFound />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
