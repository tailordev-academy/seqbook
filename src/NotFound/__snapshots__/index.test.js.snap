// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="NotFound"
>
  <div
    className="container-fluid"
  >
    <h1>
      Page Not Found
    </h1>
  </div>
</div>
`;
